src/main/java/duke/command/Command.java

```diff
@@ -5,8 +5,20 @@
 import duke.task.TaskList;
 
 public abstract class Command {
+    /**
+     * Executes command.
+     *
+     * @param tasks Current TaskList.
+     * @param storage Current Storage.
+     * @throws DukeException If invalid input.
+     */
     public abstract void execute(TaskList tasks, Storage storage) throws DukeException;
 
+    /**
+     * Returns true if it is an ExitCommand.
```

Comment:
Perhaps you could mention what returning true would do to the program? To make clearer the use of this method!